data/src/client.rs

```diff
@@ -367,6 +372,9 @@ impl Client {
                     if contains("multi-prefix") {
                         requested.push("multi-prefix");
                     }
+                    if contains("draft/read-marker") {
+                        requested.push("draft/read-marker");
+                    }
 
```

Comment:
Do we need to think about chunking this request? We've quite a few requested capabilities now